apps/pano/src/utils/index.ts

```diff
@@ -79,11 +79,16 @@ export function validateURL(url: string) {
     return true;
   }
 }
+
+export function validate(value: unknown): value is string {
```

Comment:
let's update this to accept the length as an optional argument (default: 0) and compare the trimmed length with that argument.

Reply:
Let's do this with a fast-follow PR.